README.md

```diff
@@ -11,4 +20,18 @@ VITE_UPLOAD_PRESET='YOUR-UNSIGNED-UPLOAD-PRESET'
 
 # Run
 
-`npm run dev`
+From your terminal, in the **react-pa** folder, run: 
+
+* `npm i`
+* `npm run dev` 
+
+Then, click the localhost link to open the app in your browser.
```

Comment:
"or open your browser and navigate to localhost:3000 or to the port displayed in the terminal"

Reply:
Addressed in a suggestion further down.